mozdownload/scraper.py

```diff
@@ -668,7 +667,7 @@ def get_build_info(self):
             self.logger.info('Selected build: build%d' %
                              (self.build_index + 1))
 
-    def get_build_info_for_version(self, version, build_index=None):
+    def get_build_info_for_version(self):
         url = urljoin(self.base_url, self.candidate_build_list_regex)
```

Comment:
Hm, now I see why you are removing those parameters. The version is directly used as `self.version` in `candidate_build_list_regex`, so passing it through doesn't make sense. While looking at especially this method I would say lets kill it and integrate its code to `get_build_info()`.